src/dymaptic.GeoBlazor.Core/Components/Widgets/BasemapLayerListWidget.cs

```diff
@@ -0,0 +1,85 @@
+﻿using dymaptic.GeoBlazor.Core.Components.Widgets.LayerList;
+using Microsoft.AspNetCore.Components;
+using Microsoft.JSInterop;
+using System.Text.Json.Serialization;
+
+
+namespace dymaptic.GeoBlazor.Core.Components.Widgets;
+/// <summary>
+///     The ListItem class represents one of the operationalItems in the LayerListViewModel. In the LayerList widget UI, the list item represents a layer displayed in the view. It provides access to the associated layer's properties, allows the developer to configure actions related to the layer, and allows the developer to add content to the item related to the layer.
```

Comment:
This looks like the wrong description for this class.

Reply:
updated to include basemap and later to include reference basemap